docs/src/pages/getting-started/support/support.md

```diff
@@ -0,0 +1,144 @@
+
+The current status of each Material-UI framework version is:
+
+- Material-UI v4 (hooks) **[Active]**: Latest version in active development.
+- Material-UI v3 (browser version support): Only handle security fixes.
+- Material-UI v2 (never existed).
+- Material-UI v1 (rewrite): Inactive.
+- Material-UI v0.x: Inactive.
+
+For teams and organizations that require additional support for any framework version, Material-UI has [options available](#enterprise).
+
+## Professional help (Premium)
+
+### Enterprise
+
+Accelerate your company’s application development with a secure, long-term solution that’s optimized for enterprise and backed by Material-UI experts.
+
+- **Long Term Support.** ([LTS](https://en.wikipedia.org/wiki/Long-term_support)) provides developers and organizations with additional stability and reliability in the applications they’re developing, including guaranteed support level agreements (SLAs) and advisory services. Take the preparation time you need before upgrading to the latest active version (v4) of Material-UI or remain confident in your application’s long-term success by staying on whichever version you’re most comfortable with. LTS is available as a paid service, it extends to Material-UI v1 implementations and higher.
+- **Version Migration Assistance**
+- **Priority Fixes**
+
+For more information, [get a quote](mailto:enterprise@material-ui.com) from our team.
+
+### Advisory
+
+Supercharge your team’s development with Material-UI Advisory:
+
+#### Developer Advisory
+
+Everything your team needs to make sure you’re using React the best way as you evaluate and build on Material-UI.
+
+- Unlimited technical support covering Material-UI usage of the API, component design, performance, and architectural best practices.
+- A private, dedicated Slack channel with your team and Material-UI. You get access to our pool of Solutions Architect.
+- An initial walkthrough of your architecture and development workflow.
+- Guidance on best practices that help you take full advantage of Material-UI and prevent avoidable mistakes.
+- An early look at Material-UI features in the design and development phase.
+Content, tools, and support that can help you evangelize and teach React across your company.
+
+[Get a quote](mailto:advisory@material-ui.com)
+
+#### Enterprise Advisory
+
+A complete support package, for teams using Material-UI for critical business applications and organizations deploying Material-UI across multiple teams.
+
+- Includes "Developer Advisory" scaled to your entire scope of work.
+- A designated Technical Account Manager who understands your business requirements and proactively makes sure you’re getting everything you need.
+- A designated Solutions Architect who works with your team through every phase of development, on-line.
+- Regular, scheduled reviews.
+- Prioritized assistance to help shepherd your open-source pull requests through the review process.
```

Comment:
```suggestion
```

Reply:
What issue do you see in this value proposition?

Reply:
The idea is that the "developer" offering focuses on making each dev get the most of the library while the "enterprise" offering focuses on making each organization the most of the library. In this case, what I had in mind is the following. Let's say an organization need something to be changed but doesn't want to wait for us to prioritize it, if it aligns with our direction, they could dedicate internal resources to the problem while we make sure we are not a bottleneck to make the changes land.

Ionic doesn't seem to have a similar value proposition. Maybe it's implicitly provided in the Enterprise offering. Under the "Priority Fixes" label.

Ok, I think that we can remove this dot.